lib/src/reducers/load_dna_file_reducer.dart

```diff
@@ -43,6 +43,8 @@ ${util.stack_trace_message_bug_report(stack_trace)}''';
         '\n* error type:    ${error.runtimeType}'
         '\n* error message: ${error.toString()}'
         '\n$hline'
+        '\nIf the file is imported from cadnano, use the opton of importing cadnano v2.'
+        '\nImporting a .json file is not allowed when selecting the "Open" option. Use a file with the .sc extension.'
```

Comment:
This is a bit confusing because the same error message happens when dragging a `.json` file onto the browser. So they may not have selected File-->Open to see this error message.